NEWS.md

```diff
@@ -1,7 +1,7 @@
 # rlang (development version)
 
 * `format_error_bullets()` now adds green tick bullets for elements
-  named `"v"` (@rossellhayes).
+  named `"v"` (@rossellhayes) and line breaks for elements named `" "` (#1130).
```

Comment:
Start with an empty line?

I think we also need some way to display a unindented line.

Reply:
You can include a `\n` character in the bullet to start on an empty line. I'll add examples to showcase the difference.